man/systemd.link.xml

```diff
@@ -115,6 +115,20 @@
           property <varname>DEVTYPE</varname>.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Property=</varname></term>
+        <listitem>
+          <para>A whitespace-separated list of udev property name with its value after a equal
+          (<literal>=</literal>). If multiple properties are specified, the test results are ANDed.
+          If the list is prefixed with a "!", the test is inverted.</para>
+
+          <para>Example: if a .network file has the following:
+          <programlisting>Property=ID_MODEL_ID=6864 ID_REVISION=0400</programlisting>
+          then, the .link file matches only when an interface has both
+          <literal>ID_MODEL_ID=6864</literal> and <literal>ID_REVISION=0400</literal>.
```

Comment:
What about quoting and properties with "special characters", including whitespace? Or are udev properties guaranteed to contain no whitespace?